src/main/java/github/scarsz/discordsrv/objects/threads/ChannelTopicUpdater.java

```diff
@@ -59,4 +60,18 @@ public void run() {
             }
         }
     }
+
+    private String applyLegacyPlaceholders(String text) {
+        String[] legacyPlaceholders = {"playercount","playermax","totalplayers","uptimemins","uptimehours","uptimedays",
+                "motd","serverversion","freememory","usedmemory","totalmemory","maxmemory","freememorygb","usedmemorygb","totalmemorygb","maxmemorygb","tps"};
+        for (String legacyPlaceholder : legacyPlaceholders) {
+            if (text.contains(legacyPlaceholder))
+                DiscordSRV.info("Found legacy placeholder " + legacyPlaceholder + " in " + text + ". This should be replaced with %server_" + legacyPlaceholder + "%");
```

Comment:
- This is bad as a info message (could be triggered frequently depending on activity of a server), 
- This check doesn't have % signs at the end or beginning, 
- That array could be made a variable in the class instead of in the method to save it from initiating it every time. 

We should skip 'deprecating' placeholders at this stage of v1, they should keep working like they do now